haskell-src/Concordium/Types/Updates.hs

```diff
@@ -907,6 +915,7 @@ getUpdatePayload spv =
         21 | GASRewardsVersion1 <- gasRewardsVersionFor cpv -> GASRewardsCPV2UpdatePayload <$> get
         22 | isSupported PTFinalizationCommitteeParameters cpv -> FinalizationCommitteeParametersUpdatePayload <$> get
         23 | isSupported PTValidatorScoreParameters cpv -> ValidatorScoreParametersUpdatePayload <$> get
+        24 | supportsProtocolLevelTokens spv -> CreatePLTUpdatePayload <$> get
```

Comment:
I think we also have a new chain parameter version for P9. I'd align the check with the other `isSupported` checks here depending on `cpv`.

Reply:
I can introduce another chain parameter version for P9, but currently we don't have one.
Do we expect any new parameters?

Reply:
I'm also not sure. But would be good to check.